contracts/CO30_Felix_Safety.tem_kujari/edit_me/description.ext

```diff
@@ -0,0 +1,10 @@
+// Mission settings - title, description, author etc.
+OnLoadName = "Safety";
+OnLoadMission = "Contractors provide security for local operations.";
+author = "Felix de Jong";
```

Comment:
Why do you keep removing the `, Synixe Contractors`

Reply:
I don't like it - looks too long and decentralizes the author of the mission, plus it would be even longer if more than one person makes the mission. It's the loading screen anyway so I doubt it matters, does anyone read it these days?